### PR TITLE
Update test dependencies

### PR DESCRIPTION
**Pull request by NeonDaniel**
Update test dependencies to address test failure https://github.com/OpenVoiceOS/ovos-plugin-manager/actions/runs/6635328844/job/18026087764?pr=187
Related to default configuration change https://github.com/OpenVoiceOS/ovos-config/releases/tag/V0.0.11
Refactor to run unit tests on changed requirements to catch invalid dependency changes

### CODE REVIEW

File: requirements/test.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+pytest
+pytest-timeout
+pytest-cov
+neon-lang-plugin-libretranslate
```
Comment (JarbasAl) [inline requirements/test.txt:4]: is this needed? it is no longer a default anywhere is it?

Comment (ChanceNCounter) [reply requirements/test.txt:4]: If that’s what superseded it, not needed. If that’s another name for the import, still needed. Hard to tell from a search.

Comment (JarbasAl) [reply requirements/test.txt:4]: thats just the entrypoint definition, not related to this

Comment (NeonDaniel) [reply requirements/test.txt]: > is this needed? it is no longer a default anywhere is it?

Just copied it from the GHA definition; lets see if tests still pass without it

**Pull request merged**